### PR TITLE
Feature/se 302.parity coverage scanner python

### PR DESCRIPTION
**Pull request by slacki123**
# Pull Request Checklist

- [x] Read the [contributing guidelines](../docs/CONTRIBUTING.md)
- [x] Tests pass
- [x] Raised the PR against the `develop` branch

# Description of the PR

Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

File: sdk/features/unit/dummyfiles/empties/empty_annotations2.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import unittest
+
+from features.lusid_feature import lusid_feature
+
+
+class Empties2Tests(unittest.TestCase):
```
Comment (rizwansaeed) [inline sdk/features/unit/dummyfiles/empties/empty_annotations2.py:6]: why is this needed ie. what's different vs `EmptiesTests`?

File: sdk/features/unit/dummyfiles/multidecorator/multidecorator_annotations.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import unittest
+
+from parameterized import parameterized
+
+from features.lusid_feature import lusid_feature
+
+
+class Valid2Tests(unittest.TestCase):
```
Comment (rizwansaeed) [inline sdk/features/unit/dummyfiles/multidecorator/multidecorator_annotations.py:8]: what's the significance of the '2' in Valid2Tests?

File: sdk/features/unit/dummyfiles/multidecorator/multidecorator_annotations2.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import unittest
+
+from parameterized import parameterized
+
+from features.lusid_feature import lusid_feature
+
+
+class Valid2Tests(unittest.TestCase):
```
Comment (rizwansaeed) [inline sdk/features/unit/dummyfiles/multidecorator/multidecorator_annotations2.py:8]: as above, why duplicated?

**Review by rizwansaeed — CHANGES_REQUESTED** [review]

File: sdk/features/unit/dummyfiles/empties/empty_annotations2.py (hunk 1 above)
Comment (slacki123) [reply sdk/features/unit/dummyfiles/empties/empty_annotations2.py:6]: Again to test that it targets all files, and that doesn't throw any errors in the event of some files being unannotated

File: sdk/requirements.dev.txt [hunk 4]
```diff
@@ -1 +1,2 @@
-parameterized==0.7.*
\ No newline at end of file
+parameterized==0.7.*
+lusidfeature>=0.1.3
```
Comment (rizwansaeed) [inline sdk/requirements.dev.txt:2]: remove lusidfeature and add following as first line

`-r requirements.txt`

File: sdk/requirements.txt [hunk 5]
```diff
@@ -4,4 +4,5 @@ python_dateutil >= 2.5.3
 setuptools >= 21.0.0
 urllib3 >= 1.15.1
 pytz >= 2019.1
-requests >= 2.21.0
\ No newline at end of file
+requests >= 2.21.0
+lusidfeature>=0.1.3
```
Comment (rizwansaeed) [inline sdk/requirements.txt:8]: remove the version from lusidfeature

**Review by rizwansaeed — CHANGES_REQUESTED** [review]

**Review by rizwansaeed — APPROVED** [review]
